### DISCUSSION TITLE
Error: An POSIX error occurred in the XDMF writer

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#674] [Q&A]
I can run successfully by using `mpirun -np 48 ./SeisSol_Release_dhsw_6_elastic parameters.par ` in cluster, 
but when I use the same command to run a example in my computer, I meet the following problem, does any kown why this happen?
thanks in advance！
![image](https://user-images.githubusercontent.com/104721358/191748151-5acdc06d-e47b-4655-a496-5e5d99d2bc21.png)

**Comment by Thomas-Ulrich** [comment]
Do you have 48 cores on your computer?

**Reply by Thomas-Ulrich** [reply]
Maybe try to clean the output folder from its content?
(Doesn't make sense, but I m just trying to interpret the error message)

**Comment by smallriver666** [comment]
> mpirun -np 48 ./SeisSol_Release_dhsw_6_elastic parameters.par

I am sorry,  I didn't express it clearly，I run it by using `mpirun -np 4 ./SeisSol_Release_dhsw_6_elastic parameters.par ` on my computer, and I meet the same problem, and  I have 4 cores on my computer.
